CMakeLists.txt

```diff
@@ -1,14 +1,22 @@
 cmake_minimum_required(VERSION 3.5)
+
 # silence RPATH cmake warning
 set(CMAKE_MACOSX_RPATH 1)
 set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 
+# Use vcpkg submodule toolchain by default
+set(VCPKG_OVERLAY_TRIPLETS ${CMAKE_CURRENT_SOURCE_DIR}/msvc)
```

Comment:
Will this work okay on non-msvc systems?

Reply:
This is temporary. We'll probably have the triplet files in the root dir or else a custom folder.

I was thinking of adding a "platform" dir and we could push in all the build stuff into that. It's quite busy atm with existing ones e.g Android, Bela, Emscripten, mingw64, mingw64-linux, msvc.